projects/go-tester/browserslist

```diff
@@ -8,4 +8,4 @@
 last 2 versions
 Firefox ESR
 not dead
-not IE 9-11
\ No newline at end of file
+IE 9-11
```

Comment:
Do we need to specify `9-11` here or can we just do `11`?

Reply:
Not sure, I was just doing this for testing purposes and would be dependent upon the implementor to do. I bet we can just do `IE 11`